Utopia/modules/modules_dna_rna.py

```diff
@@ -0,0 +1,74 @@
+def validation(*seqs):
+    dna_bases = {"A", "a", "T", "t", "C", "c", "G", "g"}
+    rna_bases = {"A", "a", "U", "u", "C", "c", "G", "g"}
+
+    results = []
+    for seq in seqs:
+        sequence_set = set(seq)
+
+        if sequence_set.issubset(dna_bases):
+            results.append("DNA")
+        elif sequence_set.issubset(rna_bases):
+            results.append("RNA")
+        else:
+            results.append("NotNA")
+
+    return results
+
+
+def reverse(*seqs):
+    return [seq[::-1] for seq in seqs]
+
+
+def transcribe(*seqs):  # мы не транскрибируем  из РНК ДНК
+    transcribe_d = {
+        "T": "U",
+        "A": "A",
+        "G": "G",
+        "C": "C",
+        "t": "u",
+        "a": "a",
+        "g": "g",
+        "c": "c",
+    }
+
+    out_seqs = []
+
+    for seq in seqs:
+        out_seq = []
+        for nucl in seq:
+            out_seq.append(transcribe_d.get(nucl, ""))
+        out_seqs.append("".join(out_seq))
```

Comment:
Можно использовать более просто способ с методом replace). В Вашем варианте большая часть словаря не несет какого-то значимого  (по типу А меняем на А), поэтому могу предложить заменять конкретно только тиминовые нуклеотиды.
```suggestion
     seq.replace("T", "U").replace("t","u")
```